3.2/root/usr/bin/run-mongod-pet

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash
+#
+# Run mongod as part of PetSet.
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+source ${CONTAINER_SCRIPTS_PATH}/common.sh
+
+function usage() {
+  echo "You must specify the following environment variables:"
+  echo "  MONGODB_USER"
+  echo "  MONGODB_PASSWORD"
+  echo "  MONGODB_DATABASE"
+  echo "  MONGODB_ADMIN_PASSWORD"
+  echo "  MONGODB_KEYFILE_VALUE"
+  echo "  MONGODB_REPLICA_NAME"
+  echo "Optional variables:"
+  echo "  MONGODB_SERVICE_NAME (default: mongodb)"
+  echo "MongoDB settings:"
+  echo "  MONGODB_NOPREALLOC (default: true)"
```

Comment:
Config file can contain it (it takes effect only if different storage engine is specified or data from old engine are present).

My question was mainly about if to print it in 'help' (this code is not shared with anyone else)?